fleet/backend/pktvisor/endpoint.go

```diff
@@ -53,22 +20,15 @@ func viewAgentBackendHandlerEndpoint(pkt pktvisorBackend) endpoint.Endpoint {
 			return "", errors.Wrap(errors.ErrUnauthorizedAccess, err)
 		}
 
-		if backend.HaveBackend(req.id) {
-			bk, err := Handlers()
-			if err != nil {
-				return nil, err
-			}
-			return bk, nil
-		} else {
-			return nil, errors.ErrNotFound
+		bk, err := Handlers()
```

Comment:
I would expect this to be `pkt.handlers()` - a private method of pktvisorBackend now